DataFormats/Detectors/TRD/src/RawData.cxx

```diff
@@ -41,6 +41,19 @@ void buildTrackletHCHeaderd(TrackletHCHeader& header, int detector, int rob, int
   buildTrackletHCHeader(header, sector, stack, layer, side, chipclock, format);
 }
 
+uint32_t getHCIDFromTrackletHCHeader(const TrackletHCHeader &header)
+{
+  return header.layer + header.stack * constants::NLAYER + header.supermodule * constants::NLAYER * constants::NSTACK; 
```

Comment:
I think a HCID should also contain the side. Otherwise it would be the detector ID.

Reply:
yes it should